src/bacnet/rpm.c

```diff
@@ -689,27 +694,45 @@ void rpm_ack_object_property_process(
                 &rp_data->array_index);
             if (len <= 0) {
                 /* malformed */
```

Comment:
I've bumped into a false positive result here: when the code parses the last byte in the specific RPM response (last loop iteration, apdu[0] = 0x1F, apdu_len = 1, closing tag), it falsely recognizes it as a "malformation" and executes a callback with INVALID_TAG inside. 

Please check the [bip-simulator-rpm-invalid-tag.pcapng.gz](https://github.com/user-attachments/files/17006731/bip-simulator-rpm-invalid-tag.pcapng.gz) for a full message. For "real-time debugging" you can also try this BACnet Thermostat simulator (https://github.com/Kretiss/BACnet-simulator, `pip3 install bacpypes asyncore`, `python3 rooms_simulator.py -a 192.168.0.2/24 -p 47808`), and the issue will be easily reproduced with it if you do ReadPropertyMultiple on its object analog-input:10.